src/style.css

```diff
@@ -0,0 +1,130 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+  width: 200px;
+  margin: 0 auto;
+  padding: 29px 14px 10px;
+}
+
+.card__photo {
+  margin: 0 auto;
+  margin-bottom: 45px;
+  display: block;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin: 0;
+  margin-bottom: 7px;
+  font-size: 12px;
+  line-height: 15.47px;
+  font-weight: 500;
+}
+
+.card__code {
+  display: block;
+  margin-bottom: 15px;
+  font-weight: 400;
+  font-size: 11px;
+  line-height: 12.89px;
+}
+
+.wrap {
+  display: flex;
+}
+
+.wrap__stars {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  width: 15px;
+  height: 15px;
+}
+
+.wrap__stars:nth-child(-n + 4) {
```

Comment:
add styles for each modifier, so that you could change modifier in html, and number of active stars would change accordingly